.gitignore

```diff
@@ -1,3 +1,4 @@
 .envrc
 .DS_Store
-.pytest_cache
\ No newline at end of file
+.pytest_cache
+migrations
```

Comment:
why are migrations in git ignore?

Reply:
removed migrations from gitignore and added the migrations